client/store/actions.js

```diff
@@ -15,94 +15,90 @@ function clearStore({ commit }) {
 }
 
 function thingyTileAdd({ commit, state }, quantifiedThingy) {
-  if(!state.oodlet.id) { return false; }
-  commit('thingyTileAdd', quantifiedThingy);
+  if (!state.oodlet.id) {
+    return;
+  }
 
-  oodletApi.update(state.oodlet).then(response => {
-    commit('oodletSet', response.body);
+  commit('thingyTileAdd', quantifiedThingy);
+  oodletApi.update(state.oodlet).then(oodlet => {
+    commit('oodletSet', oodlet);
   });
 }
 
 function quantifiedThingyChange({ commit, state }, payload) {
-  if(!state.oodlet.id) { return false; }
+  if (!state.oodlet.id) {
+    return;
+  }
+
   if (payload.qty === 0) {
     commit('quantifiedThingyRemove', payload.id);
-  }
-  else {
-    commit('quantifiedThingyUpdate', payload);
+    return;
   }
 
-  oodletApi.update(state.oodlet).then(response => {
-    commit('oodletSet', response.body);
+  commit('quantifiedThingyUpdate', payload);
+  oodletApi.update(state.oodlet).then(oodlet => {
+    commit('oodletSet', oodlet);
   });
 }
 
 function historyOodletLoad({ commit, state }, historyOodlet) {
-  if(!state.oodlet.id) { return false; }
+  if (!state.oodlet.id) {
+    return;
+  }
+
   oodletApi.update({
-      id: state.oodlet.id,
-      quantifiedThingies: historyOodlet.quantifiedThingies
-    })
-    .then(response => {
-      commit('oodletSet', response.body);
-    });
+    id: state.oodlet.id,
+    quantifiedThingies: historyOodlet.quantifiedThingies
+  }).then(oodlet => {
+    commit('oodletSet', oodlet);
+  });
 }
 
 function oodletLoad({ commit, state }) {
-  oodletApi.active(state.oodler.oodler).then(response => {
-    if(response.body) {
-      commit('oodletSet', response.body);
-    }
-  });
+  oodletApi.active(state.oodler.oodler).then(oodlet => commit('oodletSet', oodlet));
```

Comment:
Actually the rest is inconsistent with this one 🤣 
This is how it's intended:
```javascript
oodletApi.update(state.oodlet)
    .then(oodlet => commit('oodletSet', oodlet));
```
Good catch ⚡️